ProjFS.Mac/PrjFSKextTests/VnodeCacheTests.mm

```diff
@@ -136,20 +136,26 @@ - (void)testTryInsertOrUpdateEntry_ExclusiveLocked_ReturnsFalseWhenFull {
 static void AllocateCacheEntries(uint32_t capacity, bool fillCache)
 {
     s_entriesCapacity = capacity;
-    s_entries = static_cast<VnodeCacheEntry*>(calloc(s_entriesCapacity, sizeof(VnodeCacheEntry)));
+    s_entries = new VnodeCacheEntry[s_entriesCapacity];
     
-    if (fillCache)
+    vnode dummyNode;
```

Comment:
This should probably be a `static` variable so its address can't coincide with a later vnode.